src/main/resources/application-dev.yml

```
@@ -0,0 +1,31 @@
+spring:
+#  sql:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)